fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/service/GmailBackedPlatformEmailService.java

```diff
@@ -57,18 +57,20 @@ public void sendToUserAccount(final EmailDetail emailDetail, final String unenco
         	}
         	email.setFrom(authuser, authuserName);
 
-            final StringBuilder subjectBuilder = new StringBuilder().append("Fineract Prototype Demo: ").append(emailDetail.getContactName())
-                    .append(" user account creation.");
+            final StringBuilder subjectBuilder = new StringBuilder().append("Welcome ").append(emailDetail.getContactName())
```

Comment:
The subject line should be "New User Information" or 'Welcome to [Organization] Name'